packages/lesswrong/server/repos/PostsRepo.ts

```diff
@@ -1302,8 +1304,9 @@ class PostsRepo extends AbstractRepo<"Posts"> {
       AND "startTime" < NOW() + INTERVAL '5 months'
       AND "googleLocation" -> 'geometry' -> 'location' ->> 'lat' IS NOT NULL
       AND "googleLocation" -> 'geometry' -> 'location' ->> 'lng' IS NOT NULL
+      ${eventTypeFilter}
       LIMIT $1
-    `, [limit]);
+    `, params);
```

Comment:
Does any of this need to be changed?  I thought we weren't using this repo method in the new code at all?